compose-with-lightspeed.yaml

```
@@ -0,0 +1,22 @@
+services:
+  road-core-service:
+    image: quay.io/redhat-ai-dev/road-core-service:latest # dclint disable-line service-image-require-explicit-tag
```

```suggestion
    image: quay.io/redhat-ai-dev/road-core-service:rcs-06302025-rhdh-1.6 # dclint disable-line service-image-require-explicit-tag
```

safer tag, it is currently identical to latest but there is a potential latest breaks down the line